tests/e2e/k8s/util/kind.go

```diff
@@ -361,6 +369,14 @@ func (c *KindCluster) CreateFromYAML(yaml, namespace string) error {
 		decoder.MutateNamespace(namespace))
 }
 
+// CreateFromFolder creates k8s objects from a yaml in a folder.
+func (c *KindCluster) CreateFromFolder(folder string) error {
```

Comment:
I think folder is a Windows term. Perhaps rename to Directory?

Reply:
Or `path`.
Would it also make sense to validate that the parameter is indeed a directory and not a file or let `DirFS` fail?